scraper/data/errors.csv

```diff
@@ -0,0 +1,4790 @@
+"Plant Datatable","errno","syscall","code","path"
+"Not Found",,,,
```

Comment:
Shouldn't we handle these "Not Found" differently in the script instead of yielding an error?

Reply:
You probably mean other error types when column data is missing. Here I handle cases when it is not even an article page by checking the parent HTML nodes. For example, if there is not element with id `#plant-datatable`, I skip the page processing and mark it in `errors.csv`. So the `errors.csv` file is there for debugging purposes only. I've analyzed the entire file and they are all listing pages e.g. https://practicalplants.org/wiki/alnus/